cmd/kubeadm/app/util/system/package_validator.go

```diff
@@ -127,7 +127,8 @@ func (validator *packageValidator) validate(packageSpecs []PackageSpec, manager
 		// Get the version of the package on the running machine.
 		version, err := manager.getPackageVersion(packageName)
 		if err != nil {
-			klog.V(1).Infof("Failed to get the version for the package %q: %s\n", packageName, err)
+			logs = append(logs, logInfo(fmt.Sprintf("Failed to get the version for the package %q: %s\n", packageName, err), 1))
```

Comment:
caller should decide about the verbosity (see above)

Reply:
so all logs generated have the same verbosity? I'm not sure we want this.

Reply:
we have only `V(1)` here, also this would considerably simplify things

Reply:
yes. we have a info and an error log :) but if just strings we would lose this.

Reply:
I mean. I can change it to just strings for everything, []string for logs []string for warning errors and []string for error errors... but that feels bad to me. (and keep in mind I was not touching errors this PR. just logs. Errors' next PR.